packages/stack/src/lib/stack-app.ts

```diff
@@ -1759,6 +1773,7 @@ export type StackClientApp<HasTokenStore extends boolean = boolean, ProjectId ex
 
     signInWithOAuth(provider: string): Promise<void>,
     signInWithCredential(options: { email: string, password: string }): Promise<KnownErrors["EmailPasswordMismatch"] | undefined>,
+    createUser(options: { email: string }): Promise<KnownErrors["UserEmailAlreadyExists"] | KnownErrors["PasswordRequirementsNotMet"] | undefined>,
```

Comment:
This function should return the generated password.

Do you have any thoughts on whether createUserWithCredential is a better name for this? Since theoretically a user could be a different type as well (OAuth, magic link, etc). Maybe @fomalhautb you have an opinion on this?